services/payment-service/tests/debug/test_minimal.py

```
@@ -0,0 +1,204 @@
+            print("\n✅ 중복 INSERT 차단 확인 (UNIQUE 정상 작동)")
+
+    await engine.dispose()
+
+
+# ── 레이어 4: FastAPI app 자체가 뜨는지 ────────────────────────
+@pytest.mark.asyncio
+async def test_레이어4_앱_헬스체크():
+    """실제 DB 연결 없이 /health 엔드포인트가 응답하는지 확인."""
+    import os
+
+    os.environ["INTERNAL_SERVICE_TOKEN"] = "test-token"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "INTERNAL_SERVICE_TOKEN"

(S105)